sympy/sets/handlers/issubset.py

```diff
@@ -24,35 +25,78 @@ def is_subset_sets(a, b): # noqa:F811
     if (b.right_open and not a.right_open and fuzzy_bool(Eq(a.end, b.end))):
         return False
 
+
 @dispatch(Interval, FiniteSet)  # type: ignore # noqa:F811
 def is_subset_sets(a_interval, b_fs): # noqa:F811
     # An Interval can only be a subset of a finite set if it is finite
     # which can only happen if it has zero measure.
     if fuzzy_not(a_interval.measure.is_zero):
         return False
 
+
 @dispatch(Interval, Union)  # type: ignore # noqa:F811
 def is_subset_sets(a_interval, b_u): # noqa:F811
-    if all(isinstance(s, (Interval, FiniteSet)) for s in b_u.args):
-        intervals = [s for s in b_u.args if isinstance(s, Interval)]
-        if all(fuzzy_bool(a_interval.start < s.start) for s in intervals):
-            return False
-        if all(fuzzy_bool(a_interval.end > s.end) for s in intervals):
-            return False
-        if a_interval.measure.is_nonzero:
-            no_overlap = lambda s1, s2: fuzzy_or([
-                    fuzzy_bool(s1.end <= s2.start),
-                    fuzzy_bool(s1.start >= s2.end),
-                    ])
-            if all(no_overlap(s, a_interval) for s in intervals):
-                return False
+    unk = False
+    fin = []
+    remain = a_interval
+    for i in b_u.args:
+        if isinstance(i, FiniteSet):
+            fin.append(i)
+            continue
+        if not isinstance(i, (Union, Interval)):
+            return
```

Comment:
actually, we should mark unk=True and continue because the answer might resolve as True without these objects